aten/src/ATen/ConjugateFallback.cpp

```diff
@@ -31,6 +31,9 @@ TORCH_LIBRARY_IMPL(aten, Conjugate, m) {
   m.impl("resolve_conj", torch::CppFunction::makeFallthrough());
   m.impl("resolve_neg", torch::CppFunction::makeFallthrough());
 
+  // See test_layout_check_for_primal_with_conj_bit in test_autograd.py
+  m.impl("_has_same_storage_numel", torch::CppFunction::makeFallthrough());
```

Comment:
If you have this one, you most likely also want it for neg version.
And you want a special impl for zero tensor as well? That returns true like the batched version?

Reply:
> If you have this one, you most likely also want it for neg version.

Yup.

> And you want a special impl for zero tensor as well? That returns true like the batched version?

Also a good point. Zero tensors aren't landed yet though right?

Reply:
> Zero tensors aren't landed yet though right?

It was but maybe got reverted?

Reply:
It was reverted indeed.